x-pack/plugins/security_solution/cypress/tasks/alerts_detection_rules.ts

```diff
@@ -253,9 +253,14 @@ export const sortByEnabledRules = () => {
   cy.get(SORT_RULES_BTN).contains('Enabled').click({ force: true });
 };
 
+export const waitForRulesTableToShow = () => {
+  // Wait up to 5 minutes for the rules to show up as in CI containers this can be very slow
+  cy.get(RULES_TABLE, { timeout: 300000 }).should('exist');
+};
+
 export const waitForRulesTableToBeLoaded = () => {
```

Comment:
At first glance, I've the feeling this method and the above have the same objective. If so, we should consider to keep just one :)

Reply:
@MadameSheema I'd expect `waitForRulesTableToShow` to be sufficient for the vast majority of the tests. `waitForRulesTableToBeLoaded` can still be needed for some tests that check the rules table load/refetch/pagination logic. I can add this comment to the code and let's keep both for now.